backend/backend/process_client_json.py

```diff
@@ -0,0 +1,48 @@
+"""Module providing functionality to retrieve requests, parse JSON, and call on
+   roll die functionality.
+
+"""
+from cgitb import enable
+from cgi import FieldStorage
+import json
+from backend.roll_die import roll_two_dice
+
+enable()
+
+SAMPLE_DICTIONARY_OBJECT = {"key": "value"}
+
+
+def manager_function():
+    """Function to act as start point for client requests sending JSON.
+
+    Returns:
+        An ack string to inform the function terminated successfully.
+
+    """
+    print('Content-Type: text/html')
+    print()
+    client_request = interpret_request()
+    dice_result = roll_two_dice()
+    print((client_request, dice_result))
+    return "ack"
+
+
+def interpret_request():
+    """Function to take in parameters sent in client request and extract JSON.
+
+    Returns:
+        A dictionary representing the parsed JSON from the client.
+
+    """
+    print('Content-Type: text/html')
+    print()
+
+    form_data = FieldStorage()
```

Comment:
The JS request is POST now, so would this still work?

Reply:
Yes from what I remember in first year web dev, but to have a proper answer, then we need the javascript to make a request to this script